ArduPlane/Plane.h

```diff
@@ -385,8 +385,10 @@ class Plane : public AP_HAL::HAL::Callbacks {
     AP_BattMonitor battery;
 
 #if FRSKY_TELEM_ENABLED == ENABLED
+    // Inertial Navigation
+    AP_InertialNav_NavEKF inertial_nav {ahrs};
```

Comment:
Thanks for the suggestion. I'll look into ahrs.gps. For horizontal velocity, groundspeed seems to me most suitable for a copter and airspeed for a plane...